programs/castle-vault/src/instructions/rebalance.rs

```diff
@@ -60,55 +65,119 @@ pub struct Rebalance<'info> {
     //#[soteria(ignore)]
     pub port_reserve: AccountInfo<'info>,
 
+    /// CHECK: safe
```

Comment:
Can you add comments on why these are safe? eg. What lines the checks are done on